src/main/resources/db/changelog/changes/v1.0.0/sql/2021-05-13-01__create-inn-reach-location_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE inn_reach_location
+(
+    id                 UUID         NOT NULL,
+    code               VARCHAR(5)   NOT NULL UNIQUE,
```

Comment:
To enforce "lower-case alphanumeric characters" requirement you could use CHECK constraint as suggested here:
https://www.reddit.com/r/PostgreSQL/comments/2mal8j/is_there_an_alphanumeric_data_type/

Reply:
I would prefer to add such kind of validation at the REST API level, since the mod-inn-reach service is the one owner of the database, so the REST API is the only entry point.

Reply:
ok, let's do your way